buildrunner/validation/config.py

```diff
@@ -110,18 +115,19 @@ def validate_push(
                         name = f"{name}:latest"
 
                 if isinstance(push, StepPushCommitDict):
-                    names = [f"{push.repository}:{tag}" for tag in push.tags]
+                    if not push.tags:
+                        name = f"{push.repository}:latest"
+                    else:
+                        names = [f"{push.repository}:{tag}" for tag in push.tags]
```

Comment:
+BUILDRUNNER_BUILD_DOCKER_TAG or whatever it is. But it probably doesn't matter for this code, possibly.

Reply:
That is an interesting case I didn't consider. If BUILDRUNNER_BUILD_DOCKER_TAG is specified in one place of the config, but not another the config validation would miss it. Even though BUILDRUNNER_BUILD_DOCKER_TAG will ultimately get added to the blank one. This is an edge case I may make another ticket for.

Reply:
Maybe we should just base it on the repo name instead of the tag itself, since the tag may be misused sometimes (e.g. ":latest" left off but docker defaults to it in some cases, etc).